vimrc

```diff
@@ -29,6 +29,16 @@ endif
 
 filetype plugin indent on
 
+" Softtabs, 2 spaces
+set tabstop=2
+set shiftwidth=2
+set shiftround
+set expandtab
+set softtabstop=2
```

Comment:
If these were an `autocmd` for Go with `setlocal`s, all these settings would be Go-only, correct?